lib/ansible/modules/network/cnos/cnos_vlan.py

```diff
@@ -48,6 +48,8 @@
      cnos.html)
 version_added: "2.3"
 Options:
+    {}
+
     - The following is a table depicting how the overloaded variables are used
```

Comment:
These other lines will not end up in the documentation, so I believe they need removing.